<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerSupplyWidget</class>
 <widget class="QWidget" name="PowerSupplyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>21</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblState">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getStateHandler</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSimpleShape" name="ledSequence">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>10</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="colour0" stdset="0">
      <color>
       <red>5</red>
       <green>240</green>
       <blue>5</blue>
      </color>
     </property>
     <property name="colour1" stdset="0">
      <color>
       <red>252</red>
       <green>233</green>
       <blue>79</blue>
      </color>
     </property>
     <property name="colour2" stdset="0">
      <color>
       <red>240</red>
       <green>5</green>
       <blue>5</blue>
      </color>
     </property>
     <property name="colour3" stdset="0">
      <color>
       <red>20</red>
       <green>60</green>
       <blue>20</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblVoltage">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getVmon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblCurrent">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getIload</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QENumericEdit" name="txtCurrent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setReference</string>
     </property>
     <property name="precision">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="writeOnChange">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDetails">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSimpleShape" name="ledState">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Error</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
